source/reference/command/setParameter.txt

```diff
@@ -18,20 +18,40 @@ setParameter
    Replace the ``<option>`` with one of the supported
    :dbcommand:`setParameter` options:
 
-   - :parameter:`journalCommitInterval`
+   **Indexing**
+
+   - :parameter:`failIndexKeyTooLong`
+   - :parameter:`notablescan`
+
+   **Logging**
+
+   - :parameter:`logComponentVerbosity`
    - :parameter:`logLevel`
    - :parameter:`logUserIds`
-   - :parameter:`notablescan`
    - :parameter:`quiet`
+   - :parameter:`traceExceptions`
+
+   **Replication**
+
    - :parameter:`replApplyBatchSize`
    - :parameter:`replIndexPrefetch`
-   - :parameter:`syncdelay`
-   - :parameter:`traceExceptions`
-   - :parameter:`textSearchEnabled`
-   - :parameter:`sslMode`
+
+   **Security**
+
+   - :parameter:`auditAuthorizationSuccess`
    - :parameter:`clusterAuthMode`
+   - :parameter:`enableLocalhostAuthBypass`
```

Comment:
Fixed

On Wed, Mar 4, 2015 at 11:33 AM, Kay Kim <notifications@github.com> wrote:

> In source/reference/command/setParameter.txt
> <https://github.com/mongodb/docs/pull/2195#discussion_r25788680>:
>
> >     - :parameter:`clusterAuthMode`
> > +   - :parameter:`enableLocalhostAuthBypass`
>
> I think enableLocalhostAuthBypass is only set when starting up since in
> order to be able to setParameter, you'll need to be able to get in first.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mongodb/docs/pull/2195/files#r25788680>.
>



-- 
{
  name     : *"*Michael Paik*", *
  title    : "Director of Documentation",
  company  : "MongoDB, Inc.",
  web      : ["www.mongodb.org", "www.mongodb.com"]
}